GETTING_STARTED.md

```diff
@@ -41,6 +41,7 @@ class Test::Unit::TestCase
 end
 
 # Cucumber
+# env.rb (Rails example location - app/features/support/env.rb)
```

Comment:
Wow...my bad! Sorry about that. I obviously spent too long looking at code that day...

Yes, that is indeed better. Apologies for the confusion, and thanks for the reply.